scripts/build-wvr-ud.js

```diff
@@ -133,3 +145,14 @@ let timeDiff = endTime - startTime; //in ms
 timeDiff /= 1000;
 
 log(`    ${chalk.green('Usage Documentation Finished:')} ${chalk.blue(timeDiff+'s')}`);
+
+function copyFolderSync(from, to) {
```

Comment:
May want to consider using a library to avoid maintaining copy a folder. fs-extra is a common nodejs utility for this.

Reply:
fs-extra was used, and was failing on directories. That is the reason for the introduction of this new method.